src/main/resources/assessment-config/accommodation/accommodation-assessment-v3.yml

```diff
@@ -142,6 +151,8 @@ pages:
         title: "Accommodation resettlement status"
         subTitle: "Select one option."
         type: "RADIO"
+        validation:
```

Comment:
This validation message needs adding to SUPPORT_NEEDS_PRERELEASE as well for all the pathways

Reply:
Missed that on a few of the pathways - added in